docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs

```diff
@@ -110,8 +110,8 @@ void PredictSentiment(MLContext mlContext, ITransformer model)
     // </SnippetPredict>
 
     // <SnippetDisplayPredictions>
-    Console.WriteLine("Number of classes: {0}", sentimentPrediction.Prediction.Length);
-    Console.WriteLine("Is sentiment/review positive? {0}", sentimentPrediction.Prediction[1] > 0.5 ? "Yes." : "No.");
+    Console.WriteLine("Number of classes: {0}", sentimentPrediction.Prediction?.Length);
+    Console.WriteLine("Is sentiment/review positive? {0}", sentimentPrediction.Prediction?[1] > 0.5 ? "Yes." : "No.");
```

Comment:
Not related to nullable references, but I think an interpolated string reads better:

```suggestion
    Console.WriteLine($"Number of classes: {sentimentPrediction.Prediction?.Length}");
    Console.WriteLine($"Is sentiment/review positive? {(sentimentPrediction.Prediction?[1] > 0.5 ? "Yes." : "No.")}");
```